src/common.jl

```diff
@@ -9,15 +9,15 @@ function MDBValue(val::T) where {T}
     return MDB_val(Csize_t(val_size), convert(Ptr{Cvoid},pointer(val)))
 end
 
-convert(::Type{T}, mdb_val_ref::Ref{MDB_val}) where {T} = _convert(T, mdb_val_ref[])
-function _convert(::Type{String}, mdb_val::MDB_val)
+mbd_unpack(::Type{T}, mdb_val_ref::Ref{MDB_val}) where {T} = _mbd_unpack(T, mdb_val_ref[])
+function _mbd_unpack(::Type{T}, mdb_val::MDB_val) where {T <: String}
     unsafe_string(convert(Ptr{UInt8}, mdb_val.mv_data), mdb_val.mv_size)
 end
-function _convert(::Type{Vector{T}}, mdb_val::MDB_val) where {T}
+function _convert(::Type{V}, mdb_val::MDB_val) where {T, V <: Vector{T}}
```

Comment:
Is there a reason not to rename this `_convert`?

Reply:
Nope, I just somehow missed it.